### ISSUE TITLE
why isn't there a test:less than?  

### ISSUE THREAD

**Issue by DoomSquirter** [McJty/RFToolsControl#153]
Issue description: 
why isn't there a test:less than?  
I'm very new to rftools control.  I'm well versed in sfm and actual programming languages.  I've watched a few videos and read what info I can about the mod, but yeah, I'm at a loss why there doesn't seem to be a test: less than.  there's one for greater than and I understand you can use that and look at failure of that test but I can think of many situations where I'd still need a less than test.

I've looked through github issues open/closed and not seen anyone comment on this so I have to assume I am missing something very obvious.

Versions:
* Minecraft: 
1.12
* Forge: 
14.23.5.2796
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib:
mcjtylib-1.12-3.0.7
* RFTools:
rftools-1.12-7.59
* RFTools Control:
rftoolsctrl-1.12-1.9.2

Possibly incompatible mods and versions: 

Relevant logs, if any:

**Comment by McJty** [comment]
Why would you need a test:less if you have a test:greater? Just invert the input/outputs?

**Comment by DoomSquirter** [comment]
yeah, that's more or less what I mentioned.  I assumed that was the case.  I have like a form of math dyslexia if you will (lol) and literally my brain just stopped working when I hit this problem, thus my question.  but yeah, ok, I see your point then.  

I just find it weird that in every single programming language/scripting that I've ever done, there's always both < and > comparisons but found it gone here.  even if it's just for reading and being consistent for the way you're thinking the code out in your head, it just weird to me.

Just weird that I'm the first person to ask this.  thanks tho :)
